static/main.js

```diff
@@ -226,8 +224,39 @@ function setupButtons(options) {
     }
 }
 
+function configFromEmbedded(embeddedUrl) {
+    // Old-style link?
+    var params;
+    try {
+        params = url.unrisonify(embeddedUrl);
+    } catch (e) {
+        // Ignore this, it's not a problem
+    }
+    if (params && params.source && params.compiler) {
+        var filters = _.chain((params.filters || '').split(','))
+            .map(function (o) {
+                return [o, true];
+            })
+            .object()
+            .value();
+        return {
```

Comment:
should we not move this bit to components.js to leave the responsibility there?

Reply:
I'm not actually sure how to trigger this bit of code, so I can't test it...

Reply:
It's the embedded-in-iframe links of the form `https://godbolt.org/e#g:!((g:!((g:!((h:codeEdi....` etc.

Reply:
(I think)

Reply:
> It's the embedded-in-iframe links of the form `https://godbolt.org/e#g:!((g:!((g:!((h:codeEdi....` etc.

I tried both creating the link and going to resulting link, but neither triggered this code

Reply:
It's triggered it by visiting a page where there's an iframe to said link. Note that this was on sharing.js in the old version, so maybe I can leave it there for this PR, and we can think about where to put it once we merge, maybe?

Reply:
Agreed...we should probably revisit again. It might be an old link thing, which we can maybe find in the web logs if someone's hit one in the last 30d.